ga4gh/converters.py

```diff
@@ -0,0 +1,42 @@
+"""
+Provides classes that convert protocol requests to genomics data files
```

Comment:
(or just "file formats"? Although there again, is it to a file format, or some live protocol?)